### ISSUE TITLE
False positive in Style/SpaceAroundOperators cop

### ISSUE THREAD

**Issue by ghost** [sds/haml-lint#212]
Hi there,

**Rubocop** [allows space around operators for alignment](https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L1040-L1044) by default and this is the config I have set in my app too.

However, **HAML lint** complains when I do something like the following:

```haml
%td= content_tag transfer.origin.deleted?      ? :s : :span, transfer.origin.name
%td= content_tag transfer.destination.deleted? ? :s : :span, transfer.destination.name
```

Cheers.

**Comment by michaelherold** [comment]
I've checked the code that the Ruby extractor extracts. This turns out to be:

```ruby
_haml_lint_puts_0 # td
content_tag transfer.origin.deleted?      ? :s : :span, transfer.origin.name
_haml_lint_puts_1 # td/
_haml_lint_puts_2 # td
content_tag transfer.destination.deleted? ? :s : :span, transfer.destination.name
_haml_lint_puts_3 # td/
```

Because the `AllowForAlignment` setting requires the lines to be contiguous, they are being raised as a false positive.

This is a tricky one. I'm not sure if there's going to be an easy fix for it.

**Comment by easybills-admin** [comment]
Any news on this one?

**Comment by michaelherold** [comment]
I haven't been able to come up with a way to prevent this from happening.
